### PR TITLE
Add Aiven user-agent

### PR DESCRIPTION
**Pull request by eliax1996**
This PR sets the `user-agent` header properly referred to Aiven.
Also added a micro note on code formatting in the `CONTRIBUTING.md`

### CODE REVIEW

File: src/main/java/io/aiven/kafka/connect/gcs/GcsSinkTask.java [hunk 1]
```diff
@@ -33,14 +33,20 @@
...
 import com.google.api.gax.retrying.RetrySettings;
+import com.google.api.gax.rpc.FixedHeaderProvider;
 import com.google.cloud.storage.BlobInfo;
 import com.google.cloud.storage.Storage;
 import com.google.cloud.storage.StorageOptions;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 public final class GcsSinkTask extends SinkTask {
+    public static final String CONNECTOR_USER_AGENT = "kafka_connect_gcs";
```
Comment (AnatolyPopov) [inline src/main/java/io/aiven/kafka/connect/gcs/GcsSinkTask.java:44]: Does it need to be exactly like this?

File: src/main/java/io/aiven/kafka/connect/gcs/GcsSinkTask.java [hunk 2]
```diff
@@ -72,6 +78,7 @@ public void start(final Map<String, String> props) {
         this.storage = StorageOptions.newBuilder()
                 .setHost(config.getGcsEndpoint())
                 .setCredentials(config.getCredentials())
+                .setHeaderProvider(FixedHeaderProvider.create(USER_AGENT_HEADER_KEY, USER_AGENT_HEADER_VALUE))
```
Comment (AnatolyPopov) [inline src/main/java/io/aiven/kafka/connect/gcs/GcsSinkTask.java:81]: Do we need to have a way to provide a different value?

**Review by AnatolyPopov — CHANGES_REQUESTED** [review]

File: src/main/java/io/aiven/kafka/connect/gcs/GcsSinkTask.java [hunk 3]
```diff
@@ -33,14 +33,20 @@
...
 import com.google.cloud.storage.Storage;
 import com.google.cloud.storage.StorageOptions;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 public final class GcsSinkTask extends SinkTask {
+    public static final String CONNECTOR_USER_AGENT = "kafka_connect_gcs";
     private static final Logger LOG = LoggerFactory.getLogger(GcsSinkConnector.class);
+    private static final String USER_AGENT_HEADER_KEY = "user-agent";
+    private static final String USER_AGENT_HEADER_FORMAT = "%s (GPN: Aiven;) Google BigQuery Sink/%s";
```
[the remark this answers is not in the document]
Comment (eliax1996) [reply src/main/java/io/aiven/kafka/connect/gcs/GcsSinkTask.java:47]: oh you are right, I've copy pasted the name and forgot of renaming it into GCS connector

File: src/main/java/io/aiven/kafka/connect/gcs/GcsSinkTask.java (hunk 2 above)
Comment (eliax1996) [reply src/main/java/io/aiven/kafka/connect/gcs/GcsSinkTask.java:81]: wdym?

File: src/test/java/io/aiven/kafka/connect/gcs/GcsSinkTaskTest.java [hunk 4]
```diff
@@ -382,6 +383,12 @@ void setupDefaultRetryPolicy() {
 
         verify(mockedContext, never()).timeout(anyLong());
 
+        final Map<String, String> headers = storage.getOptions()
+                .getMergedHeaderProvider(new NoHeaderProvider())
+                .getHeaders();
+
+        assertThat(headers.size()).isEqualTo(1);
+        assertThat(headers.get("user-agent")).isEqualTo("Google GCS Sink/test-version (GPN: Aiven;)");
```
[the remark this answers is not in the document]
Comment (eliax1996) [reply src/test/java/io/aiven/kafka/connect/gcs/GcsSinkTaskTest.java:391]: ohh cool! Didn't know about that method! Thanks for the suggestion

**Review by AnatolyPopov — APPROVED** [review]
LGTM

**Pull request merged**